client-side/src/components/signUp/PasswordStrength.scss

```diff
@@ -0,0 +1,35 @@
+.centered-container {
+    width: 50%;
+    margin-left: 2%;
+  }
+  
+  .password-strength-meter {
+    height: 10px;
+    border-radius: 5px;
+    right: 3%;
+    background-color: #D3D3D3;
```

Comment:
colors should be saved in variable, fix all occurrences

Reply:
קשור לאפרת הרטמן